### PR TITLE
fix: add error message in GetByteCode()

### PR DESCRIPTION
**Pull request by da1suk8**
## Description

I added error message in GetByteCode().

## Motivation and context

An error occurred in GetByteCode, but the error is not returned.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [x] I have updated the documentation accordingly.
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/wasm/keeper/keeper.go [hunk 1]
```diff
@@ -876,7 +876,7 @@ func (k Keeper) GetByteCode(ctx sdk.Context, codeID uint64) ([]byte, error) {
 	var codeInfo types.CodeInfo
 	codeInfoBz := store.Get(types.GetCodeKey(codeID))
 	if codeInfoBz == nil {
-		return nil, nil
+		return nil, types.ErrNotFound
```
Comment (zemyblue) [inline x/wasm/keeper/keeper.go:879]: Please, add unittest in this line.

**Review by zemyblue — COMMENTED** [review]
Please, add changelog.

File: x/wasm/keeper/keeper.go (hunk 1 above)
Comment (da1suk8) [reply x/wasm/keeper/keeper.go:879]: I added the test.
Please could you check.

**Review by zemyblue — APPROVED** [review]

**Review by shiki-tak — APPROVED** [review]
LGTM

**Pull request merged**